raysect/core/math/function/function1d/tests/test_cmath.py

```diff
@@ -104,3 +104,19 @@ def test_atan2(self):
         for x in v:
             expected = math.atan2(self.f1(x), self.f2(x))
             self.assertEqual(function(x), expected, "Atan4Q1D call did not match reference value")
+
+    def test_erf(self):
+        v = [-1e5, -7, -0.001, 0.0, 0.00003, 10, 23.4, 1e5]
+        function = cmath1d.Erf1D(self.f1)
+        for x in v:
+            expected = math.erf(self.f1(x))
+            self.assertEqual(function(x), expected, "Erf1D call did not match reference value.")
+
+    def test_sqrt(self):
+        v = [0.0, 0.00003, 10, 23.4, 1e5]
+        function = cmath1d.Sqrt1D(self.f1)
+        for x in v:
+            expected = math.sqrt(self.f1(x))
+            self.assertEqual(function(x), expected, "Sqrt1D call did not match reference value.")
+
+        self.assertTrue(math.isnan(function(-0.1)), msg="Sqrt1D did not return nan for negative value.")
```

Comment:
This needs to be changes to detect an exception. In general NaNs are a bad idea.

Reply:
Changed to exception detection.